src/View/Widget/DateTimeWidget.php

```diff
@@ -528,6 +540,26 @@ protected function _meridianSelect($options, $context)
         return $this->_select->render($options, $context);
     }
 
+    /**
+     * Generates a timezone select
+     *
+     * @param array $options The options to generate a timezone select.
+     * @param \Cake\View\Form\ContextInterface $context The current form context.
+     * @return string
+     */
+    protected function _timezoneSelect($options, $context)
+    {
+        $options += [
+            'name' => '',
+            'val' => null,
+            'style' => ['display: none'],
+            'options' => [$options['val'] => $options['val']],
+            'templateVars' => [],
+        ];
+
+        return $this->_select->render($options, $context);
```

Comment:
Why are we making a select and not a hidden input? Seems like a hidden input is what we want here.

Reply:
I agree. How would that look like here? Using a widget in a widget?

Reply:
That's what `$this->_select` is as well. The WidgetRegistry stitches various widget dependencies together.